docs/static/azure-module.asciidoc

```diff
@@ -0,0 +1,591 @@
+[role="xpack"]
+[[azure-module]]
+=== Azure Module  [Experimental]
+experimental[]
+
+The https://azure.microsoft.com/en-us/overview/what-is-azure/[Microsoft Azure] module in Logstash helps you easily integrate your Azure
```

Comment:
Can we put some more detail on what the module does? Maybe something like the below...

"The Logstash Azure module enables you to easily monitor your Azure cloud environments and SQL DB deployments with deep operational insights across multiple Azure subscriptions. Explore the health of your infrastructure in real-time to accelerate root cause analysis and decrease overall time to resolution.

- Analyze infrastructure changes and authorization activity
- Identify suspicious behaviors and potential malicious actors
- Perform root-cause analysis by investigating user activity
- Monitor and optimize your SQL DB deployments"